### PR TITLE
added alex tool to check insensitive words in our documentation

### LINKED ISSUE #3402: insensitive words in *.md files
**Expected Behaviour**

http://alexjs.com/ is a tool to catch insensitive, inconsiderate or inappropriate writing per CLI.

Benefits are catching insensitive writing early on as well as possibly educate new contributors on that topic plus send an additional signal (to CoC etc.) that we are very committed to keep that project like that.

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by anishagg17**
Fixes #3402

Changes: 

added alex tool to check insensitive words in our documentation

### CODE REVIEW

**Comment by anishagg17** [comment]
@akshatnitd currently tests for .md files are running twice on travis ci because it is used in test script of package.json and .travis.yml file too.

Please let me know which one fits the best

File: docs/PROJECT_INFORMATION.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ SUSI.AI can be used anomymously or as a logged in user by using the account feat
 
 ### Retrieving User Chat History
 
-Whenever a user logs in he must be able to view his chat history in the chat client.
+Whenever a user logs in they must be able to view theirs chat history in the chat client.
```
Comment (akshatnitd) [inline docs/PROJECT_INFORMATION.md]: `their`

File: docs/Skill_Tutorial.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
 # SUSI.AI Skill Development Tutorial
 
-Do you want your own AI Skills for SUSI? It's surprisingly easy to add more Skills to SUSI.AI
+Do you want your own AI Skills for SUSI? It's not difficult to add more Skills to SUSI.AI
```
Comment (akshatnitd) [inline docs/Skill_Tutorial.md]: Revert this change

File: docs/Skill_Tutorial.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 
 ## Getting Started
 
-It's easy, DON'T PANIC. You don't need to be a software developer to enhance SUSI.AI
+It's uncomplicated, DON'T PANIC. You don't need to be a software developer to enhance SUSI.AI
```
Comment (akshatnitd) [inline docs/Skill_Tutorial.md]: Revert this change

File: docs/Skill_Tutorial.md [hunk 4]
```diff
@@ -60,10 +60,10 @@ Therefore, we can access every expert with the 4-tuple
...
+As you will see in the tutorial levels below, the language looks like an extremely effortless pattern-matching Question-Answer declaration. But that is the facade, it is easier for beginners but under the hood there is an expert system which is able to do planning and proving. The artificial intelligence in SUSI.AI is implemented as a theorem prover which can explore the domain of possible answers and gives one which is _true_ in SUSI's universe of knowledge.
 
 However, we had to combine the complex world of declaration-based logic programming with natural language declarations.
-We believe that the result is a easy-to-learn skill language with the ability to advance to true artificial intelligence when we reach a certain level of experience.
+We believe that the result is a uncomplicated skill language with the ability to advance to true artificial intelligence when we reach a certain level of experience.
```
Comment (akshatnitd) [inline docs/Skill_Tutorial.md]: Revert this change

File: docs/Skill_Tutorial.md [hunk 5]
```diff
@@ -73,14 +73,14 @@ In your editor window write:
 roses are red
 SUSI is a hack
 ```
-This defines one simple intent: to answer on "roses are red" the phrase "SUSI is a hack". The other lines mean:
+This defines one light intent: to answer on "roses are red" the phrase "SUSI is a hack". The other lines mean:
```
Comment (akshatnitd) [inline docs/Skill_Tutorial.md]: Revert this change

File: docs/Skill_Tutorial.md [hunk 6]
```diff
@@ -73,14 +73,14 @@ In your editor window write:
...
 * all lines starting with `#` are comment lines and are ignored.
 * all other text lines define Skills. Skills are separated by empty lines. Comment and section declaration modifiers also count as empty lines and separate Skills.
 
 Now you can test the new intent:
 * send the following query to SUSI: "roses are red"
 * SUSI.AI will answer with "SUSI is a hack".
-The Skill file is just a text file where two lines which are not separated by an empty line represent a conversation pattern.
+The Skill file is a text file where two lines which are not separated by an empty line represent a conversation pattern.
```
Comment (akshatnitd) [inline docs/Skill_Tutorial.md]: Revert this change

File: docs/Skill_Tutorial.md [hunk 7]
```diff
@@ -197,7 +197,7 @@ In this example, SUSI.AI remembers your mood and can tell you about it. The actu
 ### Tutorial Level 8: Conditions for Answers
 
 Whenever you are using variables in answers which are not set in the same rule, you should test if these variables exist and had been set.
-It is possible to add simple conditions to the answer lines:
+It is possible to add not burdensome conditions to the answer lines:
```
Comment (akshatnitd) [inline docs/Skill_Tutorial.md]: Revert this change

File: docs/Skill_Tutorial.md (hunk 6 above)
Comment (anishagg17) [reply docs/Skill_Tutorial.md]: ON reverting tests won't pass as it is insensitive in language

**Comment by anishagg17** [comment]
@akshatnitd the unresolved changes will lead to test failure as 
ON reverting tests won't pass as they are insensitive in language

File: docs/Skill_Tutorial.md (hunk 6 above)
Comment (akshatnitd) [reply docs/Skill_Tutorial.md]: In that case, I am not sure about the flexibility of the Alex tool and if we should go with it.

Comment (anishagg17) [reply docs/Skill_Tutorial.md]: This tool is just to make language more sensitive and checks for any insensitive grammar 
For this line it would serve the same purpose as without the word 'just'
